inference-engine/src/transformations/include/transformations/utils/utils.hpp

```diff
@@ -44,6 +45,15 @@ bool has_op_with_type(const std::shared_ptr<const ngraph::Function> &function) {
     return false;
 }
 
+inline std::string create_ie_output_name(const ngraph::Output<ngraph::Node>& input) {
+    const auto& prev_layer = input.get_node_shared_ptr();
+    auto out_idx = input.get_index();
+    std::string out_name = prev_layer->get_friendly_name();
```

Comment:
```suggestion
    std::string out_name = output.get_node()->get_friendly_name();
```
This must be faster

Reply:
prev_layer is also used in the next line.